e2e/cypress/test_module/cypress/features/home-page/HomePage.feature

```diff
@@ -0,0 +1,51 @@
+Feature: |User Home Page|
+  Verify the all elements of the home page is displayed correctly
+
+  Background: Before each
+    Given the user is on home page
+
+  Scenario: View Header section
+    When the user views the header
+    Then the user should see the logo, Search input, basket ahd Login button inside header
+    And the user should see the menu-list with thee items inside header
```

Comment:
`Then` should be at the end. `Then` typically describes the expected results at the end of the scenario. Also, do we need 9 and 10 lines together? I think they are similar

Reply:
Grouped Than and And in one step